pmc.py

```diff
@@ -95,4 +76,12 @@ def run(self):
 
 
 if __name__ == "__main__":
-    PainlessMouseController().run()
+    parser = ArgumentParser(prog        = 'python pmc.py',
+                            description = strings.SUMMARY,
+                            epilog      = strings.EXAMPLES_AND_NOTES,
+                            formatter_class = RawTextHelpFormatter)
+    parser.add_argument('--speed'  , type=int, default=1, help=strings.SPEED)
+    parser.add_argument('--vim'    , action='store_true', help=strings.VIM)
+    parser.add_argument('--verbose', action='store_true', help=strings.VERBOSE)
+    args = parser.parse_args()
+    PainlessMouseController(speed=args.speed, verbose=args.verbose, vim_mode=args.vim).run()
```

Comment:
Used ArgumentParser. Generally your code was sufficient, because there are very few parameters; but if you go on and add new parameters later, then the parser will make things more convenient

Reply:
Again, please use `black` for formatting code. extra spaces don't look that good
Also what is the purpose of `prog        = 'python pmc.py'` ? 
will it work if suppose I use python3 pmc.py or python3.8 pmc.py ?

Reply:
I'm unfamiliar with Black, so let me check and install it..

Reply:
Regarding `prog = 'python pmc.py'` - it's just a program name, it will be displayed in the help like this:

`usage: python pmc.py [-h] [--speed SPEED] [--vim] [--verbose]`

It doesn't affect the way you can run the script, it's just a helper string for explanation. Naturally, you can run it with python3.8, or python3.10, or python3.11, etc.